CMakeLists.txt

```diff
@@ -2,8 +2,14 @@ PROJECT (libsolv)
 
 CMAKE_MINIMUM_REQUIRED (VERSION 2.4)
 
+IF (MSVC)
+  SET (DISABLE_SHARED_DEFAULT ON)
+ELSE ()
+  SET (DISABLE_SHARED_DEFAULT OFF)
+ENDIF ()
+
 OPTION (ENABLE_STATIC "Build a static version of the libraries?" OFF)
-OPTION (DISABLE_SHARED "Do not build a shared version of the libraries?" OFF)
+OPTION (DISABLE_SHARED "Do not build a shared version of the libraries?" DISABLE_SHARED_DEFAULT)
```

Comment:
Why are we making this change? Doesn't it make more sense just to set the option accordingly for an individual build?

Reply:
Yep, you're right.